xtext/start.sh

```diff
@@ -1,6 +1,23 @@
-
 #! /bin/bash
 
+# clean any previous editor instances and builds
+rm -rf ${ES_BUILD_LOCATION}/*
+echo Old builds cleaned.
+
+rm -rf ${ES_UPLOAD_LOCATION}/*
+echo Old uploads cleaned.
+
+find ${ES_DEPLOY_FILE_LOCATION}/ -type f -not -name 'ROOT.war' -delete
+find ${ES_DEPLOY_FILE_LOCATION}/ -mindepth 1 -type d -not -name 'ROOT' -exec rm -rf {} \+ 
+echo Old editor instances cleaned.
+
+# setup cron job to periodically stop the server
+if [ ! -z "${XTEXT_ES_STOP_CRON_TIME}" ]; then
+    (crontab -l; echo "${XTEXT_ES_STOP_CRON_TIME}" echo Scheduled shutdown triggered. "&&" killall -u root) | crontab -
```

Comment:
Does this repeatedly add the same line to crontab every time the container is restarted? Would it make more sense to do this in the Dockerfile so it only happens once?

Reply:
Yes agree, I'll move it.

Reply:
Moved in dd85e790d7052db22578f8d7085d3d5d760664f9

Reply:
The added environment variables are so that the paths used by the editor server and its scripts are specified in one location rather than hard-coded into each one, perhaps could have been done another way.